src/charm.py

```diff
@@ -152,12 +141,16 @@ def _configure(self, _):
                 logger.error(f"Failed to replan; pebble plan: {container.get_plan().to_dict()}")
                 raise e
```

Comment:
Not sure. This may be a question for Michele given [this commit](https://github.com/canonical/prometheus-k8s-operator/commit/2913c91505653e96c8958457243c74f8f03d08ec).

Reply:
@mmanciop any reason not to change this raise to block & return?